wormchain/proto/wormhole/tx.proto

```diff
@@ -159,3 +163,15 @@ message MsgMigrateContractResponse {
   bytes data = 1;
 }
 // this line is used by starport scaffolding # proto/tx/message
+
+message MsgInPlaceUpgrade {
+  // Sender is the actor that signs the messages
+  string signer = 1;
+  // Possible set of in place upgrades to perform
+  MsgSetTokenFactoryPfmDefaultParams set_tokenfactory_pfm_default_params = 2;
```

Comment:
Can `MsgSetTokenFactoryPfmDefaultParams set_tokenfactory_pfm_default_params = 2` be remove and replaced with `bytes vaa = 2`? Then, msgServer InPlaceUpgrade parses and verifies VAA, and performs the appropriate execution based on the Action?